### PR TITLE
feat: replace mongo client

### PR DESCRIPTION
**Pull request by reinaldooli**
Using the previous client it was impossible to connect with mongo v5+ version.

### CODE REVIEW

**Comment by pableeee** [comment]
LGTM

File: middleware/middleware.go [hunk 1]
```diff
@@ -74,7 +74,7 @@ func Version(v string) func(http.Handler) http.Handler {
 func Logging(logger logrus.FieldLogger) func(http.Handler) http.Handler {
 	return func(next http.Handler) http.Handler {
 		return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-			reqID := uuid.NewString()
+			reqID := uuid.New().String()
```
Comment (dreis0) [inline middleware/middleware.go:77]: Why is this needed?

Comment (miguelreiswildlife) [inline middleware/middleware.go:77]: Why is this needed?

Comment (pableeee) [reply middleware/middleware.go:77]: I believe its just because of the uuid version. Looks like `NewString` is not available.

Comment (reinaldooli) [reply middleware/middleware.go:77]: Indeed, the `NewString` is not available and it was making tests to fail.
